superset/views/core.py

```diff
@@ -105,8 +106,12 @@
     get_viz,
 )
 
+
 config = app.config
 CACHE_DEFAULT_TIMEOUT = config.get("CACHE_DEFAULT_TIMEOUT", 0)
+SQLLAB_QUERY_COST_ESTIMATE_TIMEOUT = config.get(
```

Comment:
Can you add this config var to the config file and document what it does there?

Reply:
+1